nodes.adoc

```diff
@@ -0,0 +1,1163 @@
+$ oc adm cordon <node_name>
+----
+
+
+[source,terminal]
+----
+$ oc adm drain <node_name> --force=true
+----
+
+
+[source,terminal]
+----
+$ oc delete node <node_name>
+----
+
+
+[source,terminal]
+----
+$ oc create -f 05-worker-kernelarg-selinuxoff.yaml
+----
+
+## Automatically scaling pods with the horizontal pod autoscaler
+
+[source,terminal]
+----
+$ oc edit hpa hpa-resource-metrics-memory
+----
+
+
+[source,terminal]
+----
+$ oc autoscale dc/<dc-name> \
+  --min <number> \
+  --max <number> \
+  --cpu-percent=<percent> 
+----
+
+
+[source,terminal]
+----
+$ oc autoscale rc/<rc-name> 
+  --min <number> \
+  --max <number> \
+  --cpu-percent=<percent> 
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc autoscale dc/image-registry --min 3 --max 7 --cpu-percent=75
+----
+
+
+[source,terminal]
+----
+$ oc edit hpa frontend -n openshift-image-registry
+----
+
+
+[source,terminal]
+----
+$ oc autoscale dc/image-registry --min=5 --max=7 --cpu-percent=75
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f hpa.yaml
+----
+
+## Configuring an {product-title} cluster for pods
+
+[source,terminal]
+----
+$ oc create -f <file_or_dir_path>
+----
+
+
+[source,terminal]
+----
+$ oc create -f </path/to/file> -n <project_name>
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+## Placing pods on specific nodes using node selectors
+
+[source,terminal]
+----
+$ oc patch MachineSet <name> --type='json' -p='[{"op":"add","path":"/spec/template/spec/metadata/labels", "value":{"<key>"="<value>","<key>"="<value>"}}]'  -n openshift-machine-api
+----
+
+
+[source,terminal]
+----
+$ oc patch MachineSet abc612-msrtw-worker-us-east-1c  --type='json' -p='[{"op":"add","path":"/spec/template/spec/metadata/labels", "value":{"type":"user-node","region":"east"}}]'  -n openshift-machine-api
+----
+
+
+[source,terminal]
+----
+$ oc edit MachineSet abc612-msrtw-worker-us-east-1c -n openshift-machine-api
+----
+
+
+[source,terminal]
+----
+$ oc label nodes <name> <key>=<value>
+----
+
+
+[source,terminal]
+----
+$ oc label nodes ip-10-0-142-25.ec2.internal type=user-node region=east
+----
+
+## Using device plug-ins to access external resources with pods
+
+[source,terminal]
+----
+$ oc create -f devicemgr.yaml
+----
+
+## Including pod priority in pod scheduling decisions
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+## Providing sensitive data to pods
+
+[source,terminal]
+----
+$ oc create -f <filename>
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc delete secret <secret_name>
+----
+
+
+[source,terminal]
+----
+$ oc annotate service <service_name> service.alpha.openshift.io/serving-cert-generation-error-
+----
+
+
+[source,terminal]
+----
+$ oc annotate service <service_name> service.alpha.openshift.io/serving-cert-generation-error-num-
+----
```

Comment:
Requires a declarative alternative

Reply:
I do not know how to create this scenario. Any thoughts?
https://docs.openshift.com/container-platform/4.7/cicd/builds/troubleshooting-builds.html#builds-troubleshooting-service-certificate-generation_troubleshooting-builds

Reply:
I see now that this one is not suitable to be done declaratively given that the docs is showing how to clean up the resources.